zeus/tests/test_poll_views.py

```diff
@@ -0,0 +1,164 @@
+from django.test import TestCase
+
+import datetime
+import os
+
+from helios.models import Election, Poll, Voter
+from heliosauth.models import User
+from zeus.models import Institution
+from zeus.tests.utils import SetUpAdminAndClientMixin
+
+
+class TestElectionBase(SetUpAdminAndClientMixin, TestCase):
```

Comment:
Since you only have one test class (`TestPollViews`), no need to have these helper methods in a separate class. It can all just live in `TestPollViews`.